test/pkg/test/framework.go

```diff
@@ -528,12 +563,13 @@ func (env *Framework) DeleteAllFrameworkTracedTargetGroups(ctx aws.Context) {
 					g.Expect(err.(awserr.Error).Code()).To(Equal(vpclattice.ErrCodeResourceNotFoundException))
 				}
 			}
-		}).WithTimeout(360 * time.Second).Should(Succeed())
+		}).WithTimeout(8 * time.Minute).Should(Succeed())
 
 	}
 	env.TestCasesCreatedServiceNames = make(map[string]bool)
 }
 
+// TODO: Change this function name to GetVpcLatticeServiceDnsFromHttpRoute()
```

Comment:
change function name as TODO? why not now?

Reply:
Ok, will do name change in current or next PR